typings/styled-components.d.ts

```diff
@@ -207,9 +90,12 @@ export function injectGlobal(strings: TemplateStringsArray, ...interpolations: S
 
 export const ThemeProvider: ThemeProviderComponent<any>;
 
+export class StyleSheetManager extends React.Component<{ sheet: ServerStyleSheet }, any> { }
+
 export class ServerStyleSheet {
-  collectStyles(children: ReactElement<any>): ReactElement<any>
-  getStyleTags(): string
+  collectStyles(children: ReactElement<any>): StyleSheetManager
+  collectStyles(tree: any): StyleSheetManager;
```

Comment:
ReactNode is defined as `type React.ReactNode = string | number | boolean | {} | React.ReactElement<any> | (string | number | boolean | any[] | React.ReactElement<any>)[]
`
seems to accept anything anyway